docs/preprocessing.md

```diff
@@ -0,0 +1,13 @@
+## Executing fMRIPrep
+In order to run multiple sessions in parallel for a single subject, a trick needs to be applied to avoid several jobs to work simultaneously on the same file thus compromising it.
```

Comment:
```suggestion
Because *fMRIPrep* creates a single anatomical reference for all sessions, we will need first to run with ``--anat-only`` so that the results get cached and *fMRIPrep* processes do not run into race conditions.

Similarly, when running *fMRIPrep* on a dataset with many sessions, the safest approach is to submit one job per session.
```